src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+
+const sortList = (array) => {
+  const newList = array.sort(
+    (a, b) =>
+      Number(b.getAttribute('data-salary').replace('$', '').replace(',', '')) -
+      Number(a.getAttribute('data-salary').replace('$', '').replace(',', '')),
+  );
+
+  document.querySelector('ul').innerHTML = '';
+  newList.forEach((person) => document.querySelector('ul').appendChild(person));
+};
+
+function getEmployees(array) {
+  const newArray = array.map((person) => ({
+    name: person.textContent,
+    position: person.getAttribute('data-position'),
+    salary: Number(person.getAttribute('data-salary')),
```

Comment:
Ensure that the `data-salary` attribute is always a valid number string. If there's a possibility of invalid data, consider adding error handling or validation.